node/src/command.rs

```diff
@@ -301,7 +301,7 @@ pub fn run() -> Result<()> {
 				let id = ParaId::from(para_id);
 
 				let parachain_account =
-					AccountIdConversion::<polkadot_primitives::v2::AccountId>::into_account_truncating(&id);
```

Comment:
I wonder why they have done versioning here. Does it mean there is a context where v2 should still be the case?

Reply:
v2 was removed upstream and v4 apeared

Reply:
Parachain host api v4
https://github.com/paritytech/polkadot/pull/6885